server/routes/caseNotes/caseNotesController.test.ts

```diff
@@ -98,6 +98,7 @@ describe.each([
         communityApiService.getServiceUserByCRN.mockResolvedValue(deliusServiceUserFactory.build())
         await request(app)
           .get(`/${user.userType}/referrals/${sentReferral.id}/case-notes`)
+          .expect(200)
```

Comment:
I think the commit message here (show full SU name on case notes) isn't right for the code change (adding an expectation for a 200 status code)

Reply:
Think that was introduced by a rebase. I've dropped the commit.